http/src/main/resources/reference.conf

```diff
@@ -542,6 +546,7 @@ persistentDisk {
   defaultDiskType = "pd-standard"
   defaultBlockSizeBytes = 4096
   zone = "us-central1-a"
+  defaultGalaxyNFSDiskSizeGB = 300
```

Comment:
Nit: It'd be more consistent to pick the convention either of line 267 or line 549 (unit in comment vs embedded in name) and go with it.